src/pages/Profile.jsx

```diff
@@ -89,7 +89,7 @@ const Profile = () => {
       <Box sx={{ display: 'flex', flexDirection: 'column', alignItems: 'center', gap: '20px' }}>
         <Typography sx={{ fontWeight: 'bold', fontSize: '18px' }}>Profile Information</Typography>
         <Typography>
-          {client ?? (
+          {contact ?? (
```

Comment:
The bug where it breaks the webpage when clicking on the contact URL is not really a problem with this PR. A full fix for this should be ready when PR #394 gets merged into to Development and this branch getting updated.